src/common/ZulipTextButton.js

```diff
@@ -0,0 +1,109 @@
+/* @flow strict-local */
+import React, { useMemo } from 'react';
+import { View } from 'react-native';
+
+import type { LocalizableText } from '../types';
+import { BRAND_COLOR, createStyleSheet } from '../styles';
+import { Label } from '.';
+import Touchable from './Touchable';
+
+// TODO(?): Support, e.g., "small", "medium", "large" if it makes
+// sense. It probably doesn't.
+type Variant = 'standard';
+
+const styleSheetForVariant = (variant: Variant) =>
+  createStyleSheet({
+    // See https://material.io/components/buttons#specs.
+    touchable: {
+      height: 36,
+      padding: 8,
+      minWidth: 64,
+    },
+
+    // Chosen because of the value for this distance at
+    // https://material.io/components/banners#specs.
+    leftMargin: {
+      marginLeft: 8,
+    },
+    rightMargin: {
+      marginRight: 8,
+    },
+
+    // `Touchable` only accepts one child, so make sure it fills the
+    // `Touchable`'s full area and centers everything in it.
+    childOfTouchable: {
+      flex: 1,
+      alignItems: 'center',
+      justifyContent: 'center',
+    },
+
+    text: {
+      color: BRAND_COLOR,
+      textAlign: 'center',
+      textAlignVertical: 'center',
+    },
+  });
+
+type Props = $ReadOnly<{|
+  /**
+   * See note on the `Variant` type.
+   */
+  variant?: Variant,
+
+  /**
+   * Give correct in-between space for a right-aligned set of buttons.
+   */
+  leftMargin?: true,
```

Comment:
Huh, my auto-wrap settings would prefer that it break between "of" and "buttons". But I can resist that without lots of effort and get it on one line; done.

Reply:
Got it, thanks! I now have those same settings; looks like I'd hard-coded that extension's settings to use 70, instead of adding `editor.rulers` and having it consult those